src/plugins/azgenerator/CodeModelAz.ts

```diff
@@ -71,6 +71,7 @@ export interface CodeModelAz
     CommandGroup: OperationGroup;
     CommandGroup_Name: string;
     CommandGroup_Help: string;
+    CommandGroup_DefaultName: string;
```

Comment:
where is it set?

Reply:
it already set in here https://github.com/Azure/autorest.az/blob/master/src/plugins/azgenerator/CodeModelAzImpl.ts#L467 Just only in CodeModelAzImpl.ts not in  CodeModelAz.ts

Reply:
oh, ok, so it was implemented, just not exposed